catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -168,6 +180,10 @@ public Object visit(PropertyIsBetween filter, Object extraData) {
 
     @Override
     public Object visit(PropertyIsEqualTo filter, Object extraData) {
+        if (filter.getExpression1() instanceof PropertyIsFuzzyFunction) {
```

Comment:
Agreed.  For clarity I've moved this logic into the CswQueryFactory since the filter should be built before being visited.